apps/src/templates/teacherDashboard/EditSectionForm.jsx

```diff
@@ -46,10 +47,10 @@ class EditSectionForm extends Component {
     handleSave: PropTypes.func.isRequired,
     handleClose: PropTypes.func.isRequired,
     isSaveInProgress: PropTypes.bool.isRequired,
-    textToSpeechUnitIds: PropTypes.arrayOf(PropTypes.number).isRequired,
-    lessonExtrasAvailable: PropTypes.func.isRequired,
+    assignedUnitLessonExtrasAvailable: PropTypes.bool.isRequired,
```

Comment:
It looks like we have several fields related to the assigned unit.  Do we have a "unit" object in this area that we can use to hold all of these values?  (If not, we don't necessarily have to create one for this change but something to think about for the future.)